src/sensors/bmi160sensor.cpp

```diff
@@ -25,13 +25,19 @@
 #include "network/network.h"
 #include "GlobalVars.h"
 
+#define SENSORS_GRAVITY_EARTH 9.80665
```

Comment:
this should probably be in a different file so it's not duplicated

Reply:
Could move to `sensors.h`, I guess.

Reply:
or to constants.h